apps/web/src/app/(dashboard)/admin/stats/page.tsx

```diff
@@ -43,12 +60,30 @@ export default async function AdminStatsPage() {
         <CardMetric icon={UserPlus2} title="App Version" value={`v${process.env.APP_VERSION}`} />
       </div>
 
-      <div className="mt-16 grid grid-cols-1 gap-8 md:grid-cols-1 lg:grid-cols-2">
+      {/* TODO: remove grid and see something */}
+      <div className="mt-16 gap-8">
+        <div>
+          <h3 className="text-3xl font-semibold">User metrics</h3>
+
+          <div className="mb-8 mt-4 grid flex-1 grid-cols-1 gap-4 md:grid-cols-2">
+            <CardMetric
+              icon={File}
+              title="Users with document in the last month"
```

Comment:
Maybe this?

> Users who sent one or more documents in the past month

Reply:
thank you, i had no idea what to write there